server/api/v1/event/[eventSlug]/subscription/mine.put.ts

```
@@ -0,0 +1,58 @@
+    eventSlug,
+  });
+
+  if (!subscription) {
+    throw createError({
+      statusCode: 404,
+      statusMessage: "Subscription not found",
+    });
+  }
+
+  subscription.lastQuestionnairePreScreening = {
+    formResponseId: new Types.ObjectId(parsed.data.formResponseId) as unknown as {
```

ESLint

[error] Replace `parsed.data.formResponseId` with `⏎······parsed.data.formResponseId,⏎····`

(prettier/prettier)